site/content/entry/yuzu-migration/index.md

```diff
@@ -0,0 +1,81 @@
++++
+date = "2019-10-04T23:51:00+05:30"
+title = "yuzu - The Migration"
+author = "CaptV0rt3x"
+forum = 150132
++++
+
+A very good day to you all ***yuz-ers*** out there! 
+We are currently in the process of merging both the Canary and Nightly versions of yuzu into a single release channel.
+Jump right in to find out more about this!
+<!--more-->
+***
+
+## Announcement
+
+<article class="message">
+<div class="message-header is-dropdown">
+<p>We are merging yuzu nightly and yuzu canary releases into a single "yuzu" release!</p>
+</div>
+</article>
+
+Don't fret, we are just doing some reorganization to make things easier for both the users and the developers.
+Today marks the retirement of both yuzu nightly and yuzu canary release channels.
+From tomorrow, yuzu will have only a single release channel, dubbed just "yuzu".
+
+Most of you might be thinking, "why are they doing this?" or "what does this mean for the users?".
+Let's go back in time for a bit to understand the "why?" part.
+
+## A big "Why?"
+
+When yuzu was first forked from Citra and was setting up shop as a separate project, it borrowed a few things from Citra (apart from the code).
+In those, were the general ideas about build releases, build generating infrastructure, and release channel naming convention.
+For a good while, these ideas served the purpose - that is to generate builds and distribute them, but it was high time for some improvements.
+
+In the initial days of yuzu development, due to the large list of missing/unknown functionality, we had to add `asserts` (or `assertions`) wherever necessary in the codebase. 
+Assertions are statements used to test assumptions made by the programmers.
+This allowed us to understand what functionality we were missing and which games used those, if we needed to test them.
+
+The philosophy behind the nightly channel was to have a release channel with asserts enabled, so that the developers keep getting data on what games were missing what functionality.
+But having yuzu assert and crash while playing a game rendered it unusable for normal users.
+To overcome this, we decided to release the canary builds with a specific `IGNORE ASSERTS` code included.
+
+This made yuzu ignore all those assertions and allowed normal users to test or play games on it.
+But as yuzu progressed further, the canary builds gained increased popularity among the users as the nightly builds were absolutely unusable.
+Because of this the developers too had to concentrate more on the canary builds, so that they could get users to test their code and thus improve yuzu.
```

Comment:
I'd change `Because of this` to `As a result,` and also remove `too` after `developers`